buildSrc/src/main/kotlin/plugin/FeatureModulePlugin.kt

```diff
@@ -24,7 +24,17 @@ class FeatureModulePlugin : BasePlugin() {
         get() = {
             implementation(
                 Library.daggerHiltAndroid,
-                Library.coroutines
+                Library.coroutines,
+                Library.constraintLayout,
+                Library.material,
+                Library.navigationFragmentKtx,
+                Library.navigationUiKtx,
+                Library.coreKtx,
+                Library.appCompat,
+                Library.recyclerView,
+                Library.coroutines,
```

Comment:
You have coroutines declared twice

Reply:
I'll take one out, must have been an error